### PR TITLE
Update Methods Section

### PR DESCRIPTION
**Pull request by cgreene**
This is a work in progress - filing so that @danich1 might be able to start on some of the content identified here.

### CODE REVIEW

**Comment by cgreene** [comment]
Ok @danich1  - I removed the WIP from the title because this is probably ready for a first look from you. Definitely not ready to merge though! 😁

**Review by danich1 — APPROVED** [review]
Answered/filled in the brackets with suggested changes. Will give the approval status, but won't merge until I get final approval from you.

File: content/03.methods.md [hunk 1]
```diff
@@ -1,104 +1,123 @@
...
+The number of journals returned via this method could be less than ten as multiple papers in close proximity to query article may belong to the same journal.
+Because this approach was based on paper proximity, we could return the articles that led to each journal being returned.
+However, journals that publish more papers are more frequently recommended in this framing.
+
+For the journal-centric approach, we identified the ten most similar journals by constructing a journal representation in the same embedding space.
+We computed journal centroids as the average embedding of all published papers in the journal.
+We then project a query article into the same space and return the ten closest journal centroids by Euclidean distance.
+This technique guaranteed that at least ten distinct journals were returned and prevented journals that publish many papers from being heavily overrepresented.
+
+In both cases, we optimized [WHAT NEEDED TO BE OPTIMIZED? THERE AREN'T A LOT OF FREE PARAMETERS HERE] the approaches by via 10-fold cross validation.
```
[the remark this answers is not in the document]
Comment (cgreene) [reply content/03.methods.md:111]: I thought we just set this to 10?

[the remark this answers is not in the document]
Comment (danich1) [reply content/03.methods.md:115]: Ack you are correct. Will update to reflect changes.

File: content/03.methods.md [hunk 2]
```diff
@@ -1,105 +1,133 @@
...
+The extracted text is then fed into our word2vec model to construct a document embedding representation.
+We pass this representation onto our journal venue classifiers to recommend journals based on the ten closest neighbors of individual papers as well as journal centroids.
+[THIS USES KD TREES IN SKLEARN TO MAKE IT SCALABLE AND EFFICIENT AND I THINK THEY HAD TO BE SHARDED TO WORK GIVEN THE AVAILABLE RAM THOUGH THAT MIGHT NOT BE TRUE ANYMORE - GET THE DETAILS FROM DONGBO :) ]
+
+Accompanying these recommendations, we also provide a neural network derived visualization of our training set and the article's position within it.
+We used SAUCIE [@doi:10.1101/2020.03.04.975177], an autoencoder designed to cluster single cell RNA-seq data, to build a two-dimensional embedding space that could be applied to newly generated preprints without retraining, a limitation of other approaches that we explored for visualizing entities expected to lie on a nonlinear manifold.
+We trained this model [ON WORD EMBEDDINGS DIRECTLY OR PCS AND ALSO WHAT DOCUMENTS?] using a hidden size of 2, a learning rate of 0.001, lambda_b of 0, lambda_c of 0.001, and lambda_d of 0.001 for 2000 iterations.
```
Comment (danich1) [inline content/03.methods.md:129]:
```suggestion
We trained this model on document embeddings of PMCOA articles that did not contain a matching preprint version.
We used the following parameters to train the model: a hidden size of 2, a learning rate of 0.001, lambda_b of 0, lambda_c of 0.001, and lambda_d of 0.001 for 2000 iterations.
```

File: content/03.methods.md [hunk 3]
```diff
@@ -1,105 +1,131 @@
...
-This model enabled us to visualized the PMC corpus and to efficiently embed new papers and preprints within this space.
-We trained this model using a learning rate of 0.001, lambda_b of 0, lambda_c of 0.001, and lambda_d of 0.001 for 2000 iterations.
+We developed a web application to make this prediction method available for all _bioRxiv_ preprints.
+Our web application downloads a pdf version of a preprint hosted on the  _bioRxiv_ or _medRxiv_ server.
+We use pdfminer [@url:https://pdfminersix.readthedocs.io/en/latest/index.html] to extract text from the downloaded pdf.
+The extracted text is then fed into our word2vec model to construct a document embedding representation.
+We pass this representation onto our journal venue classifiers to recommend journals based on the ten closest neighbors of individual papers as well as journal centroids.
+[THIS USES KD TREES IN SKLEARN TO MAKE IT SCALABLE AND EFFICIENT AND I THINK THEY HAD TO BE SHARDED TO WORK GIVEN THE AVAILABLE RAM THOUGH THAT MIGHT NOT BE TRUE ANYMORE - GET THE DETAILS FROM DONGBO :) ]
```
Comment (dongbohu) [inline content/03.methods.md:125]:
```
We use k-d Trees in sklearn to search the papers. In order to make it more memory efficient, the k-d trees were sharded into four pieces and constructed beforehand. They are loaded into memory directly when the web server starts.
```
Do you need more details?

File: content/03.methods.md [hunk 4]
```diff
@@ -1,105 +1,133 @@
...
-We trained this model using a learning rate of 0.001, lambda_b of 0, lambda_c of 0.001, and lambda_d of 0.001 for 2000 iterations.
+### Web Application for Discovering Similar Preprints and Journals
+
+We developed a web application that identifies similar papers and journals for any _bioRxiv_ and _medRxiv_ preprint and that places the preprint into the overall document landscape.
+Our web application downloads a pdf version of a preprint hosted on the  _bioRxiv_ or _medRxiv_ server.
+We use pdfminer [@url:https://pdfminersix.readthedocs.io/en/latest/index.html] to extract text from the downloaded pdf.
+The extracted text is then fed into our word2vec model to construct a document embedding representation.
+We pass this representation onto our journal venue classifiers to recommend journals based on the ten closest neighbors of individual papers as well as journal centroids.
+[THIS USES KD TREES IN SKLEARN TO MAKE IT SCALABLE AND EFFICIENT AND I THINK THEY HAD TO BE SHARDED TO WORK GIVEN THE AVAILABLE RAM THOUGH THAT MIGHT NOT BE TRUE ANYMORE - GET THE DETAILS FROM DONGBO :) ]
```
Comment (danich1) [inline content/03.methods.md:125]:
```suggestion
We pass this representation onto our journal venue classifiers to identify journals based on the ten closest neighbors of individual papers as well as journal centroids.
Our classifiers use k-d trees in sklearn to efficiently search for similar papers and journals. 
We sharded the k-d trees into four separate pieces to make our models more memory efficient. 
We load each piece directly into memory as soon as the web server starts.
```

File: content/03.methods.md [hunk 5]
```diff
@@ -1,105 +1,135 @@
...
-When preprints had multiple versions, we used only the latest one. 
-Preprints in this snapshot were grouped into one of twenty-nine different categories by researchers submitting to _bioRxiv_.
-Each preprint was also classified either as a new result, confirmatory finding, or contradictory finding.
-Some preprints in this snapshot were withdrawn from bioRxiv: when this happens their content is replaced with the reason for withdrawal.
+When preprints had multiple versions, we used the latest one unless otherwise noted.
+Authors submitting preprints to _bioRxiv_ select one of twenty-nine different categories.
+Researchers also select an article type, which can be a new result, confirmatory finding, or contradictory finding.
+Some preprints in this snapshot were withdrawn from bioRxiv: when this happens their content is replaced with the reason for withdrawal. 
+As there were very few withdrawn preprints, we did not treat these as a special case.
+#### PubMed Central Open Access Corpus
```
Comment (cgreene) [inline content/03.methods.md:16]:
```suggestion

#### PubMed Central Open Access Corpus
```

File: content/03.methods.md [hunk 6]
```diff
@@ -1,105 +1,135 @@
...
-
-#### Dimensionality Reduction of Document Embeddings
-We used principal component analysis (PCA) [@doi:10.1111/1467-9868.00196] to project bioRxiv document vectors into a low dimensional space.
-We trained this model using scikit-learn's [@raw:scikit-learn] implementation of a randomized solver [@arxiv:0909.4061] with a random seed of 100, output of 50 principal components, and default settings for all other hyperparameters.
-For each principal component we calculated its cosine similarity with  all tokens in our word2vec model's vocabulary.
-We report the top 100 positive and negative scoring tokens in the form of  word clouds, where the size of each word corresponds to the magnitude of similarity and color represents positive (orange) or negative (blue) association.
+Following training, we generated a document vector for every article within _bioRxiv_ and PubMed Central.
+We calculated the document vector by taking the average of every token present within a given article [@arxiv:1405.4053].
+Words absent from the word2vec model were ignored.
```
Comment (cgreene) [inline content/03.methods.md:58]:
```suggestion
Words absent from the word2vec model were ignored.

```

**Review by cgreene — COMMENTED** [review]
header changes

**Comment by cgreene** [comment]
If this builds let's merge it and move on to the next one 🚀

**Pull request merged**